src/numba_stats/crystalball.py

```diff
@@ -38,11 +38,11 @@ def _log_powerlaw(z, beta, m):
 @_jit(3, narg=0)
 def _powerlaw_integral(z, beta, m):
     T = type(beta)
-    exp_beta = np.exp(-T(0.5) * beta * beta)
-    a = (m / beta) ** m * exp_beta
+    c = -T(0.5) * beta**2
```

Comment:
beta * beta is probably faster than beta ** 2, under the assumption that LLVM translates this literally, i.e. beta * beta into self-multiplication, while beta ** 2 leads to a function call to pow(x, 2).